apps/src/templates/census2017/CensusForm.jsx

```diff
@@ -63,14 +63,14 @@ class CensusForm extends Component {
   checkShowFollowUp() {
     const twentyHours = this.state.submission.twentyHours;
     this.setState({
-      showFollowUp: (twentyHours === 'Some' || twentyHours === 'All')
+      showFollowUp: (twentyHours === 'some' || twentyHours === 'all')
```

Comment:
Why not something like? `showFollowUp: (twentyHours === i18n.some())`
Not a suggestion, just curious 🙂

Reply:
I changed it so that the value is no longer the i18n string.